src/pages/en/tutorial/5-astro-api/2.md

```diff
@@ -21,9 +21,9 @@ You can create entire sets of pages dynamically using `.astro` files that export
 
 ## Create pages dynamically
 
-1. Create a new file at `src/pages/tags/[tag]`. (You will have to create a new folder.) Notice that the file name (`[tag].astro`) uses square brackets. Paste the following code into the file:
+1. Create a new file at `src/pages/tags/[tag].astro`. (You will have to create a new folder.) Notice that the file name (`[tag].astro`) uses square brackets. Paste the following code into the file:
 
-    ```astro title="src/pages/posts/tags/[tag].astro"
+    ```astro title="src/pages /tags/[tag].astro"
```

Comment:
There shouldn’t be a space in the path here, I guess?
````suggestion
    ```astro title="src/pages/tags/[tag].astro"
````